install/kubernetes/templates/_container_oci_hooks_setup.tpl

```diff
@@ -0,0 +1,19 @@
+{{- define "container.tetragon-oci-hook-setup" -}}
```

Comment:
Could we also define `extraVolumeMounts` and `resources` for this container, configurable via Helm values? Similarly like for the tetragon container.

Reply:
Done, thanks!